priv/repo/migrations/20210414003100_drop_last_seen_customer.exs

```diff
@@ -0,0 +1,9 @@
+defmodule ChatApi.Repo.Migrations.DropCusotmerLastSeen do
+  use Ecto.Migration
+
+  def change do
+    alter table(:customers) do
+      remove :last_seen
```

Comment:
we should probably also ensure that any records with a missing `last_seen_at` value get populated with whatever was in `last_seen`

Reply:
Oops, I took a pull from master and realized you have already done this!